MdeModulePkg/Bus/Pci/PciBusDxe/PciDoe.c

```diff
@@ -19,98 +19,209 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define PCI_VENDOR_ID_PCI_SIG       0x01
 #define PCI_DOE_PROTOCOL_DISCOVERY  0x00
 
-VOID
+#define POLL_INTERVAL_US  500
+
+/** Stalls the BSP for the minimum of POLL_INTERVAL_US and Timeout.
+
+   @param[in]  Timeout    The time limit in microseconds remaining for
+                          APs to return from Procedure.

```

Comment:
Why is there a reference to APs here.  I do not see MP code.

Reply:
Just a leftover comment